examples/DotNetCore/AzureFunction/FunctionApp/Startup.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.FeatureManagement;
+
+[assembly: FunctionsStartup(typeof(FunctionApp.Startup))]
+
+namespace FunctionApp
+{
+    class Startup : FunctionsStartup
+    {
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            IConfigurationRefresher configurationRefresher = null;
+
+            // Load configuration from Azure App Configuration
+            ConfigurationBuilder configurationBuilder = new ConfigurationBuilder();
+            configurationBuilder.AddAzureAppConfiguration(options =>
+            {
+                options.Connect(Environment.GetEnvironmentVariable("ConnectionString"))
+                       // Load all keys that start with `TestApp:`
+                       .Select("TestApp:*")
+                       // Configure to reload configuration if the registered key is modified
+                       .ConfigureRefresh(refreshOptions =>
+                            refreshOptions.Register("TestApp:Settings:Message", refreshAll: true)
```

Comment:
Since we know the `refreshAll` parameter is most useful when it is tracking a sentinel key, we can consider changing the key name registered with refreshAll: true from `TestApp:Settings:Message` to something like `TestApp:Settings:ConfigVersion` or `TestApp:Settings:Sentinel`.

Reply:
I was debating myself between simplicity and best practice. You have a good point about `refreshAll`. Sure, I will change back to use the sentinel key.